### ISSUE TITLE
Error when deserializing a response containing empty `fields`

### ISSUE THREAD

**Issue by andrii-bodnar** [crowdin/crowdin-api-client-go#71]
When making a call to the Crowdin API, if the response model is one of the following:

- model/projects.go: [Project](https://github.com/crowdin/crowdin-api-client-go/blob/1fac74645a6af92e874822b30e055c7af3e1f12c/crowdin/model/projects.go#L36)
- model/source_files.go: [File](https://github.com/crowdin/crowdin-api-client-go/blob/1fac74645a6af92e874822b30e055c7af3e1f12c/crowdin/model/source_files.go#L139)
- model/source_strings.go: [SourceString](https://github.com/crowdin/crowdin-api-client-go/blob/1fac74645a6af92e874822b30e055c7af3e1f12c/crowdin/model/source_strings.go#L26)
- model/tasks.go: [Task](https://github.com/crowdin/crowdin-api-client-go/blob/1fac74645a6af92e874822b30e055c7af3e1f12c/crowdin/model/tasks.go#L56)
- model/users.go: [User](https://github.com/crowdin/crowdin-api-client-go/blob/1fac74645a6af92e874822b30e055c7af3e1f12c/crowdin/model/users.go#L211)

The following error is encountered:

```console
json: cannot unmarshal array into Go struct field File.data.data.fields of type map[string]interface{}
```

By default, it is presented as an empty array in the response:

```json
{
  "fields": []
}
```

Here is an example when there is a value:

```json
{
  "fields": {
    "type": [
      "super"
    ]
  }
}
```

Basically, `fields` is a very flexible entity and it can contain any value.

Probably, this issue can be resolved by changing the type of the `Fields` field from `map[string]any` to `[]any` in all of these models.

This case should be covered by Unit tests as well.
